src/componentes/unmounted/lista-nominal/print/aux/Selecionar_status_usuario_descricao.jsx

```
@@ -0,0 +1,96 @@
+    15: estilos.tagLaranjaComIcone,
+    16: estilos.tagVermelhaComIcone
+  }
+
+  const descricao = status_usuario_descricao.find(item => item?.id_status_usuario == value)?.status_usuario_descricao
+
+  return <div style={{
+    ...styleStatus[value],
+    width: orientacao === "retrato" ? "90%" : "73%",
+    fontSize: orientacao === "retrato" && "9px"
+  }}>
+    {[13,14,16].includes(value) &&  <img src={icones.alerta_simbolo}/>}
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)